apps/wpa_supplicant/src/utils/wpa_debug_tinyara.c

```diff
@@ -0,0 +1,762 @@
+/*
```

Comment:
But all fo files in wpa_supplicant do not have Samsung license. I sent a mail for this. We need to discuss.